public/assets/i18n/translations/da.json

```diff
@@ -76,7 +76,10 @@
   "services": "Services",
   "learnmore": "Læs mere",
```

Comment:
oversættelserne til toastify skal være:
"saveeditusernotifysuccess": "The changes have not been saved",
 "saveeditusernotifyfail": "The changes were not saved",